bundles/org.openhab.binding.radiothermostat/src/main/java/org/openhab/binding/radiothermostat/internal/handler/RadioThermostatHandler.java

```diff
@@ -287,7 +285,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         // don't do anything if we are not in heat or cool mode
                         break;
                     }
-                    connector.sendCommand(cmdKey, cmdInt.toString());
+                    connector.sendCommand(cmdKey, cmdInt.toString(), DEFAULT_RESOURCE);
+                    break;
+                case REMOTE_TEMP:
+                    if (cmdInt != -1) {
+                        connector.sendCommand("rem_temp", cmdInt.toString(), REMOTE_TEMP_RESOURCE);
```

Comment:
```suggestion
                        QuantityType<?> quantity = ((QuantityType<Temperature>) command).toUnit(SIUnits.Celsius);
                        connector.sendCommand("rem_temp", String.valueOf(quantity.intValue()), REMOTE_TEMP_RESOURCE);
```

Please just make the recommended change.

Reply:
OK, done.